### PR TITLE
Restore Master Performance

### PR DESCRIPTION
**Pull request by fthaler**
- Small environment updates to improve CPU performance stability (minor instabilities still expected on simple stencils).
- Fixed field initialization of horizontal_diffusion and stencil_on_edges_multiplefields benchmarks.
- Updated performance references, finally based on the new perftests.

### CODE REVIEW

**Comment by fthaler** [comment]
perftest

**Comment by fthaler** [comment]
perftest

**Comment by fthaler** [comment]
perftest

**Comment by fthaler** [comment]
perftest

**Comment by fthaler** [comment]
perftest

**Comment by fthaler** [comment]
perftest

**Comment by fthaler** [comment]
perftest

**Comment by fthaler** [comment]
perftest

**Comment by fthaler** [comment]
perftest

**Comment by fthaler** [comment]
launch jenkins

File: tests/regression/icosahedral/stencil_on_edges_multiplefields.cpp [hunk 1]
```diff
@@ -45,14 +45,10 @@ namespace {
...
-                backend_t(),
-                TypeParam::make_grid(),
-                TypeParam::icosahedral_make_storage(edges(), in1),
-                TypeParam::icosahedral_make_storage(edges(), in2),
-                out);
-        };
+        auto comp = [grid = TypeParam::make_grid(),
+                        in1 = TypeParam::icosahedral_make_storage(edges(), in1),
+                        in2 = TypeParam::icosahedral_make_storage(edges(), in2),
+                        &out] { run_single_stage(test_on_edges_functor(), backend_t(), grid, in1, in2, out); };
```
Comment (anstaf) [inline tests/regression/icosahedral/stencil_on_edges_multiplefields.cpp:51]: Nice!

**Pull request merged**